sdk/azidentity/troubleshoot.md

```diff
@@ -0,0 +1,108 @@
+# Troubleshoot Azure Identity authentication issues
+
+`azidentity` credential types return errors when authentication fails or they are otherwise unable to authenticate.
+This troubleshooting guide describes steps you can take to resolve such errors.
```

Comment:
```suggestion
`azidentity` credential types return errors when authentication fails or they are otherwise unable to authenticate. This troubleshooting guide describes steps you can take to resolve such errors.
```